extensions/resteasy-reactive/quarkus-rest-client-microprofile/runtime/src/main/java/io/quarkus/rest/rest/client/microprofile/RestClientBuilderImpl.java

```diff
@@ -146,6 +174,16 @@ public RestClientBuilder register(Object component, Map<Class<?>, Integer> contr
     public <T> T build(Class<T> aClass) throws IllegalStateException, RestClientDefinitionException {
         RestClientListeners.get().forEach(listener -> listener.onNewClient(aClass, this));
 
+        // mstodo if config doesn't say otherwise:
+        Object defaultMapperDisabled = getConfiguration().getProperty("microprofile.rest.client.disable.default.mapper");
```

Comment:
The `String` should be extracted into a constant to avoid duplication.
Also, why do we need to extract the property twice? Is one of them MP and other Quarkus config?

Reply:
The one below is a global config property, configured via application.properties or microprofile-config.properties.
This one is per client. It's set via `RestClientBuilder#property(String, Object)`